src/main/java/org/jenkinsci/test/acceptance/selenium/Scroller.java

```diff
@@ -126,8 +142,12 @@ public void disableStickyElements() {
         final JavascriptExecutor executor = (JavascriptExecutor) driver;
         try {
             executor.executeScript(disableStickyElementsJs);
-        } catch (UnhandledAlertException ignored) {
+        } catch (UnhandledAlertException unexpected) {
             // if we're in the process of navigating but an alert is in the way we can't run JS
+            LOGGER.log(
```

Comment:
may as well log the failure.  the comment may be wrong as this is called by some code outside this class, but this was existing.